assembl/static2/js/app/components/common/description/description.jsx

```diff
@@ -0,0 +1,18 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+export type Props = {
+  /** Description content */
+  children: React.Node,
+  /** Optional added classes */
+  className: ?string
```

Comment:
From your tests, className is not required. The flow definition should be `className?: ?string` in this case.